test/typescript-support/src/index.ts

```diff
@@ -14,6 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+// eslint-disable-next-line import/no-unresolved
```

Comment:
**question:** Can you please add a description why we need this and why it is not resolved by the plugin? We know it is resolved otherwise checks won't pass. 
I suspect that the lint task fails on CI because it runs whereas the bpmn-visualization package has not been bundled, so the resolution fails.

https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-unresolved.md

Reply:
```suggestion
// eslint-disable-next-line import/no-unresolved -- I don't know why it's failed. I propose to find later.
```

Reply:
Do you reproduce the problem locally after having building the bpmn-visualization npm package (npm pack)?

I would prefer "the bpmn-visualization package may not have been built prior running eslint (it happens when running GitHub Actions)"

Reply:
Indeed, it's the problem. So, I will change the suggestion ^^